browser/resources/settings/brave_icons.html

```diff
@@ -113,3 +113,18 @@
     </defs>
   </svg>
 </iron-iconset-svg>
+<iron-iconset-svg name="brave_lion" size="16">
```

Comment:
This is promising but I'm confused about a couple of things:
- brave lion refers to the brave logo. Did you mean Brave Leo for the design system code name?
- These are 16x16 icons but all brave design system icons are 24x24 at https://www.figma.com/file/g8Z0q6TMPYDq6zXh9Y7LWD/%F0%9F%8C%8E-Universal?node-id=61%3A30937

Reply:
yes, this is suppose to be `brave_leo`

Reply:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/8665427/169929185-44326444-1dc5-4e08-9983-6c6285def4de.png">